src/ncurses_ui.cc

```diff
@@ -474,7 +474,14 @@ void NCursesUI::draw(const DisplayBuffer& display_buffer,
     while (line_index < dim.line + line_offset)
     {
         m_window.move_cursor(line_index++);
-        m_window.draw(m_palette, DisplayAtom("~"), face);
+        if (m_padding_fill)
+        {
+            ColumnCount column_index = 0;
+            while (column_index++ < dim.column)
```

Comment:
Here we should add `m_padding_char.length()` instead of just incrementing the current column